<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="soccf-tr5-001" name="parallel-ui-tests(failed)(failed)(failed)(failed)" tests="3" failures="3" timestamp="8 Oct 2014 12:02:54 GMT" time="144.867" errors="0">
  <testcase name="allTests" time="21.841" classname="teammates.test.cases.ui.browsertests.StudentProfilePageUiTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;Your profile picture has been saved successfully&amp;gt; but was:&amp;lt;&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<Your profile picture has been saved successfully> but was:<>
	at teammates.test.pageobjects.AppPage.verifyStatus(AppPage.java:926)
	at teammates.test.cases.ui.browsertests.StudentProfilePageUiTest.testActions(StudentProfilePageUiTest.java:106)
	at teammates.test.cases.ui.browsertests.StudentProfilePageUiTest.allTests(StudentProfilePageUiTest.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- allTests -->
  <testcase name="testAll" time="115.973" classname="teammates.test.cases.ui.browsertests.InstructorFeedbackResultsPageUiTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
	at teammates.test.cases.ui.browsertests.InstructorFeedbackResultsPageUiTest.testDownloadAction(InstructorFeedbackResultsPageUiTest.java:430)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackResultsPageUiTest.testAll(InstructorFeedbackResultsPageUiTest.java:57)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testAll -->
  <testcase name="testAll" time="135.334" classname="teammates.test.cases.ui.browsertests.AdminHomePageUiTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;Your profile picture has been saved successfully&amp;gt; but was:&amp;lt;&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<Your profile picture has been saved successfully> but was:<>
	at teammates.test.pageobjects.AppPage.verifyStatus(AppPage.java:926)
	at teammates.test.cases.ui.browsertests.AdminHomePageUiTest.testCreateInstructorAction(AdminHomePageUiTest.java:285)
	at teammates.test.cases.ui.browsertests.AdminHomePageUiTest.testAll(AdminHomePageUiTest.java:72)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testAll -->
</testsuite> <!-- parallel-ui-tests(failed)(failed)(failed)(failed) -->
